cedar-policy/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- `PartialResponse` api has changed significantly
```

Comment:
```suggestion
- for the `partial-eval` experimental feature: `PartialResponse` api has changed significantly
```

Also, for the last few releases we've been putting changes to experimental apis at the bottom of the corresponding changelog section, regardless of how major they are